packages/ember/addon/instance-initializers/sentry-performance.ts

```diff
@@ -24,20 +27,23 @@ export async function instrumentForPerformance(appInstance: ApplicationInstance)
   const sentryConfig = config.sentry;
   const tracing = await import('@sentry/tracing');
 
-  const router = appInstance.lookup('service:router');
+  const idleTimeout = config.transitionTimeout || 15000;
```

Comment:
Damn that’s a high idle timeout. Any reason we bumped it?

Reply:
I've been thinking about this a little bit because I think our idleTimeout of 2000 might be too short. If you have a long loading query in your model in Ember, it will block renders until complete instead of return interleaved calls for async components (not that it can't, just that it doesn't with ember-data by default).

The main difference here is that Ember has a `routeDidChange` unlike react, which will always get called and thus call `finish` so that it won't capture extraneous events, which makes it a lot safer to increase idleTimeout to capture more of the navigation.